extensions/@shopgate-theme-config/extension-config.json

```diff
@@ -8,7 +8,26 @@
       "path": "frontend/index.js"
     }
   ],
-  "configuration": {},
+  "configuration": {
```

Comment:
Currently it's not possible to create different configurations for the two themes.

Reply:
Split config into `themeGmd` and `themeIos`